etc/profile-a-l/clipit.profile

```diff
@@ -34,16 +38,26 @@ nodvd
 nogroups
 noinput
 nonewprivs
+noprinters
 noroot
 nosound
 notv
 nou2f
 novideo
 protocol unix
 seccomp
+tracelog
 
 disable-mnt
+private-bin clipit,xdotool
 private-cache
 private-dev
+private-lib
```

Comment:
```suggestion
#private-lib
```

Wouldn't this break everything with the default firejail.config settings?

Reply:
@rusty-snake [on Dec 10](https://github.com/netblue30/firejail/pull/5521#discussion_r1045138911):

> Why?

```console
$ firejail --quiet --noprofile --private-lib echo hello
Error: private-lib feature is disabled in Firejail configuration file /etc/firejail/firejail.config
```

Reply:
This would mean if you harden firejail by disabling stuff in firejail.config would break half of your profiles? No of course not, if an command is disabled in firejail.config (or at compile time?) we ignore it in profiles to not cause unnecessary breakage. On the commandline however, where the user explicitly requested it, we error out.

Reply:
@rusty-snake [on Dec 10](https://github.com/netblue30/firejail/pull/5521#discussion_r1045139469):

> This would mean if you harden firejail by disabling stuff in firejail.config
> would break half of your profiles? No of course not, if an command is
> disabled in firejail.config (or at compile time?) we ignore it in profiles to
> not cause unnecessary breakage. On the commandline however, where the user
> explicitly requested it, we error out.

I see, thanks.  This discrepancy makes trying to debug things on the
command-line a bit more confusing, but indeed, I suppose that it makes sense
for usability.

Printing a warning when ignoring a disabled command could be useful (to make
the user aware that it is being ignored), but anyway, that's outside the scope
of this PR.

Reply:
I am aware of only one downside of `private-lib`: it can slow down creation of the sandbox. Yes its disabled by default in /etc/firejail/firejail.config. And yes, these lines are ignored in profile files in that case. Wouldn't make much sense to comment the option IMO. Unless it breaks the app, obviously, which isn't happening here with clipit.